docs/code-search/code-navigation/private-maven-repository-configuration-scip-java.mdx

```diff
@@ -0,0 +1,85 @@
+### Configuring Maven to Use a Private Repository for SCIP-JAVA
```

Comment:
Thanks for the feedback, Anton! You're right that consistency is important. However, this documentation is primarily focused on the auto-indexing process, with scip-java used as an example to demonstrate how to configure Maven with a private repository. Given that, do you think it would be clearer to frame the doc around auto-indexing in general, rather than focusing specifically on scip-java?